doc/api/events.markdown

```diff
@@ -10,7 +10,8 @@ is opened. All objects which emit events are instances of `events.EventEmitter`.
 You can access this module by doing: `require("events");`
 
 Typically, event names are represented by a camel-cased string, however,
-there aren't any strict restrictions on that, as any string will be accepted.
+there aren't any strict restrictions on that, as any string or symbol will be
+accepted.
```

Comment:
@trevnorris Sort of. Because `b` coerces to `[object Object]`, emitting on *any* object will fire the same event as `b`.

```js
const EE = require('events');
const e = new EE();
e.on({ foo: 42 }, function() { console.log('hi'); });
e.emit({ bar: 43 }); // hi
e.emit({ baz: 44 }); // hi
```

Symbols don't get coerced for property identifiers though, so they're distinct here I think.

```js
const EE = require('events');
const e = new EE();
const a = Symbol('foo');
e.on(a, function() { console.log('hi'); });
e.emit(Symbol('foo')); // no hi
```